Gemfile

```diff
@@ -72,7 +72,7 @@ gem "rubyzip",                        "~>1.2.2",       :require => false
 gem "rugged",                         "~>0.27.0",      :require => false
 gem "snmp",                           "~>1.2.0",       :require => false
 gem "sqlite3",                        "~>1.3.0",       :require => false
-gem "sys-filesystem",                 "~>1.2.0",       :require => false
+gem "sys-filesystem",                 "~>1.2.0"
```

Comment:
Why are we removing `:require => false`? Ideally we would require on demand to avoid eager loading it

Reply:
I can put it back, but I thought the plan was to use it elsewhere eventually. Also, it means explicitly loading it for the specs, which I thought was ugly.

Reply:
@bdunne What are your thoughts here?

Reply:
Yeah, I also prefer delay requiring whenever possible.